nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -56,8 +56,10 @@ public final class GenericsChecks {
   static final Supplier<Type> JSPECIFY_NULLABLE_TYPE_SUPPLIER =
       Suppliers.typeFromString("org.jspecify.annotations.Nullable");
 
+  private final Map<Tree, Map<Type, Type>> inferredTypes = new HashMap<>();
+
   /** Do not instantiate; all methods should be static */
-  private GenericsChecks() {}
+  public GenericsChecks() {}
```

Comment:
If we're just making this public, I think we can delete it (the public no-args constructor is there by default if there are no other constructors)